client/dist/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Item Details Service</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+  </head>
+  <body>
+    <h1>About this item</h1>
+    <div id="service"></div>
+  </body>
+  <script src="bundle.js"></script>
+  <script>
+  $('a').click((e) => {
```

Comment:
i assume this is just a test to see if jquery is working?

Reply:
@minmaxdata - That piece was to change css styling for the menu bar.